src/reason_parser.mly

```diff
@@ -1049,6 +1064,8 @@ conflicts.
 
 %nonassoc below_LBRACKETAT
 %nonassoc LBRACKETAT
+%nonassoc SLASHGREATERLESS
+%nonassoc GREATERLESS
```

Comment:
It's needed to solve the 74 shift/reduce I get otherwise for turning these into infix_operators. Not sure if this is correct, and I wonder if I should go for %left or %right instead. In other words: I'm lacking knowledge/experience here.